app/uk/gov/hmrc/excisemovementcontrolsystemapi/controllers/DraftExciseMovementController.scala

```diff
@@ -45,6 +45,8 @@ class DraftExciseMovementController @Inject()(
   extends BackendController(cc) {
 
   def submit: Action[NodeSeq] =
+
+  //TODO EMCS-400 - remove the validateErnInMessageAction and instead use the MessageValidation to find the correct Ern
```

Comment:
can we do this now,  so we can rid of of this todo?

Reply:
done